actionpack/lib/action_dispatch/middleware/assume_ssl.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# When proxying through a load balancer that terminates SSL, the forwarded request will appear
+# as though its HTTP instead of HTTPS to the application. This makes redirects and cookie
+# security target HTTP instead of HTTPS. This middleware makes the server assume that the
+# proxy already terminated SSL, and that the request really is HTTPS.
+module ActionDispatch
+  class AssumeSSL
```

Comment:
To fix the CI, add this class to the autoload list here https://github.com/rails/rails/blob/c5ace249e742eeb5f5aee4b4464b9c6f7a4af3eb/actionpack/lib/action_dispatch.rb#L56-L75